arch/arm/src/common/arm_createstack.c

```diff
@@ -220,7 +220,8 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
       /* Save the adjusted stack values in the struct tcb_s */
 
       tcb->adj_stack_ptr  = (uint32_t *)top_of_stack;
-      tcb->adj_stack_size = size_of_stack;
+      tcb->adj_stack_size = size_of_stack -
+          sizeof(struct tls_info_s);
```

Comment:
What if sizeof(struct tls_info_s) is not an multiple of 8 bytes.  Does it matter if the far end of the adjusted stack is not aligned properly?